### PR TITLE
Add PayPal pop-up lightbox Solves #68

### PR DESCRIPTION
**Pull request by kubami9**
## Add PayPal pop-up lightbox working like donate section on fossasia.org. This PR solves issue #68 
When you click 'Become a supporter!' button you get pop-up lightbox with PayPal donation like on https://fossasia.org/donate.html. 

I tried to make link preview but after few hours and solid headache I decided that video will be less painful.
Link to preview video: https://drive.google.com/drive/folders/1nYjhHrQSaaHjeuN1kQfyBwAQal9Zg3FV?usp=sharing

Briefly steps I did to get this effect:
1. Deleting share files from all languages in `site/data/share` and deleting all CSS styling this button. I did it to separate this button since all of the share buttons are using function opening URL in new tab which was preventing my button from firing my scripts.
2. Creating exactly same looking button but separately.
3. Creating lightbox pop-up using code delivered by @mariobehling in issue #68 and styling it.
4. Adding file donate.js from fossasia.org repository since code from the 3rd point is also from this repository (thankfully everything works fine).
5. Creating functions to open and close lightbox and adding onclick event to support button.
6. Final fixes and improvements.

### CODE REVIEW

File: site/layouts/partials/spread.html [hunk 1]
```diff
@@ -15,11 +15,95 @@ <h2 class="section-heading">{{ . }}</h2>
...
+							            <input type="hidden" name="option_amount6" value="500.00">
+							            <input type="hidden" name="option_index" value="0">
+                                        </div>
+                                  </div>
+							<input type="image" src="https://www.paypalobjects.com/en_GB/i/btn/btn_buynowCC_LG.gif" border="0" name="submit" alt="PayPal – The safer, easier way to pay online!">
+							<img alt="" border="0" src="https://www.paypalobjects.com/en_GB/i/scr/pixel.gif" width="1" height="1">
+						</form>							
+						</div>
+							<p class="lead">
+								The recommended way for donating to us is to join as a regular supporter. You can decide on if you want to pay 10 Euro or more monthly.
```
Comment (sayak119) [inline site/layouts/partials/spread.html]: @kubami9 we can use Dollar here instead of Euro. @mariobehling can you specify the amount ?

Comment (kubami9) [reply site/layouts/partials/spread.html]: Sure, I'll do it with my next PR 😄

**Comment by mariobehling** [comment]
Good job!

We need a few more changes next:
* The heart needs to be back
* On small screens the box is to much to the bottom
* The mobile version needs to be tested (buttons are moved around)
* There should be a slight gray overlay on the site


screenshot from 2017-12-14 22-01-50

**Comment by kubami9** [comment]
@mariobehling which heart? There's still heart on the button

**Comment by mariobehling** [comment]
The heart on the left hand side below the "f" Facebook button.

**Comment by mariobehling** [comment]
screenshot from 2017-12-15 14-49-58

**Pull request merged**